test/kubemark/start-kubemark.sh

```diff
@@ -102,6 +102,10 @@ function write-pki-config-to-master {
     sudo bash -c \"echo ${CA_CERT_BASE64} | base64 --decode > /home/kubernetes/k8s_auth_data/ca.crt\" && \
     sudo bash -c \"echo ${MASTER_CERT_BASE64} | base64 --decode > /home/kubernetes/k8s_auth_data/server.cert\" && \
     sudo bash -c \"echo ${MASTER_KEY_BASE64} | base64 --decode > /home/kubernetes/k8s_auth_data/server.key\" && \
+    sudo bash -c \"echo ${REQUESTHEADER_CA_CERT} | base64 --decode > /home/kubernetes/k8s_auth_data/aggr_ca.crt\" && \
+    sudo bash -c \"echo ${AGGREGATOR_CA_KEY} | base64 --decode > /home/kubernetes/k8s_auth_data/aggr_ca.key\" && \
```

Comment:
Is there a flag missing above? Do we need this line? It does not seem to be used above.

Reply:
No flag missing and you're right we could drop the line. I had it in the original to document the location of the key for purposes of cert rotation. Really has no value here for kubemark.